api/v1alpha7/openstackmachine_types.go

```diff
@@ -89,7 +89,6 @@ type OpenStackMachineSpec struct {
 	// + ---
 	// + While it's possible to have unlimited number of block devices attached to a server instance, the number is
 	// + limited to 255 to avoid rule cost exceeded error in CRD validation.
-	// +kubebuilder:validation:MaxItems=255
```

Comment:
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/1736 is an issue that I created so at some point we can re-add CEL.